frame/support/src/storage/mod.rs

```diff
@@ -519,6 +520,14 @@ pub struct PrefixIterator<T> {
 	closure: fn(&[u8], &[u8]) -> Result<T, codec::Error>,
 }
```

Comment:
I think we can also add a function `new(prefix, drain, closure)` to PrefixIterator so that ppl can implement with their custom closure if they want

Reply:
I'm suggesting stuff outside of the scope of this PR, no need to do it now in this PR